docs/src/modules/brandingTheme.ts

```diff
@@ -403,21 +403,7 @@ export const getDesignTokens = (mode: 'light' | 'dark') =>
      *  ]
      */
     applyDarkStyles(css: Parameters<ApplyDarkStyles>[0]) {
-      if ((this as Theme).vars) {
-        // If CssVarsProvider is used as a provider,
-        // returns ':where([data-mui-color-scheme="light|dark"]) &'
-        const selector = (this as Theme)
-          .getColorSchemeSelector('dark')
```

Comment:
Can you verify if this is defined in material package ? I stumbled upon this when solving another issue.

Reply:
`getColorSchemeSelector` is defined when `CssVarsProvider` is used.

I think it should be attached to the theme by `extendTheme`. I will create another issue to tackle this separately.